packages/cli/src/commands/generate/auth/templates/auth0.auth.js.template

```diff
@@ -0,0 +1,101 @@
+// Define what you want `currentUser` to return throughout your app.
```

Comment:
This may be way too much but included as an example of how to fetch userProfile now that have the access token.

Reply:
Also, I had problems testing my changes to the cli.

While I built the cli package via `yarn build:js && yarn build:clean-dist` and I see the new/updated files:

![Screen Shot 2020-07-07 at 19 26 04](https://user-images.githubusercontent.com/1051633/86855123-ba7b9280-c087-11ea-9088-a933b942c17d.png)

and when I do a 

`yarn rwt copy ../../redwoodjs/redwood`

from there and I see the files move

when i try to test with a 

`yarn rw g auth auth0`

the files in dist seem to revert to the old files and I get a file not found error for the new auth0 template

```
  ✔ Adding required web packages...
  ✔ Adding required api packages...
  ✔ Installing packages...
  ✖ Generating auth lib...
    → ENOENT: no such file or directory, open '<MYDIRPATH>/projects/redwoodjs/auth0-auth-test-app
…
    Adding auth config to web...
    Adding auth config to GraphQL API...
    One more thing...
ENOENT: no such file or directory, open <MYDIRPATH>/projects/redwoodjs/auth0-auth-test-app/node_modules/@redwoodjs/cli/dist/commands/generate/auth/templates/auth0.auth.js.template'
```

So, unfortunately these template changes haven't been tested with a new rw app. Sorry. I'll keep trying to figure out why the files revert.

Reply:
Strange, I'm having the same issue